### ISSUE TITLE
Function.explicitly_defined_type wrapper is broken

### ISSUE THREAD

**Issue by lucasduffey** [Vector35/binaryninja-api#646]
```
>>> current_function.explicitly_defined_type
Traceback (most recent call last):
  File "<input>", line 1, in <module>
  File "/home/username/binaryninja/plugins/../python/binaryninja/function.py", line 261, in explicitly_defined_type
    return core.BNHasExplicitlyDefinedType(self.handle)
AttributeError: 'module' object has no attribute 'BNHasExplicitlyDefinedType'
```

Docs: http://api.binary.ninja/binaryninja.function-module.html#binaryninja.function.Function.explicitly_defined_type (edited)

Source: https://github.com/Vector35/binaryninja-api/blob/b049ec1662df13d409d149d937fa3366c33b32a1/python/function.py#L259

**Comment by lucasduffey** [comment]
should be `BNFunctionHasExplicitlyDefinedType` according to jordan

**Comment by lucasduffey** [comment]
https://github.com/Vector35/binaryninja-api/pull/647

**Comment by psifertex** [comment]
Fixed by #647, will be in the next dev build but not triggering a specific one right now.

Thanks @lucasduffey
